tools/release.sh

```diff
@@ -87,6 +87,14 @@ if [ "$RELEASE_BRANCH" != "candidate-$version" ] ; then
 fi
 git checkout "$RELEASE_BRANCH"
 
+for pkg_dir in $SUBPKGS_NO_CERTBOT certbot-compatibility-test .
+do
+  sed -i 's/\.dev0//' "$pkg_dir/setup.py"
```

Comment:
You run this only on Linux, with GNU sed, correct? The FreeBSD version on OS X uses `-i` for something else and crashes.

Reply:
> You run this only on Linux, with GNU sed, correct? The FreeBSD version on OS X uses -i for something else and crashes.

Yeah. I recently switched to a Mac too so I've been bitten by this, but it's a problem throughout the script that I've worked around it using GNU sed from Homebrew.

Reply:
BSD sed uses `-i` to mean the same thing (edit in-place) but expects that you'll provide it with an argument. The argument, if not the empty string, is appended as an extension to the original filename and the result is created as a backup of the original file. The script is already using `sed -i` elsewhere, though, so I'd say to leave it as is for now and then maybe deal with all the uses in a separate PR.